Automattic/jetpack
# Admin Page: indicate external link for Subscribers list

## Proposed changes:

The list lives in Calypso, so we must indicate that the link will open in a new window.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings
* Search for "Subscriptions"
* Enable the module
* A new line will appear at the bottom of the card, with a link to view your subscribers.
    * An external icon should appear on the right side.
    * When you click on the link, you should be redirected to Calypso in a new tab.
